### PR TITLE
Spanish translation of Media chapter

### PR DESCRIPTION
**Pull request by garcaplay**
Makes progress on #498

Hi! Finished the translation of the Media chapter to Spanish. I had a problem with the word "viper" in the paragraph of line 81, which I consulted with some colleagues, but none of them could find what the authors meant with it. Therefore, I've left it as "viper" in the translated text as it might be some technical vocabulary that does not have a direct translation into Spanish.

### CODE REVIEW

**Comment by bazzadp** [comment]
> I had a problem with the word "viper" in the paragraph of line 81, which I consulted with some colleagues, but none of them could find what the authors meant with it. Therefore, I've left it as "viper" in the translated text as it might be some technical vocabulary that does not have a direct translation into Spanish.

I have no idea what they mean either! 😀 Surprised I didn't spot that before. Have raised #1182 for this and asked authors to comment. In meantime will review the rest of it.

File: src/content/es/2019/media.md [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+    <img src="/static/images/2019/media/fig13_project_perf_improvements_image_optimization.png" alt="Figura 13. Estimación de la mejora del rendimiento de la página tras la optimización de imagen de Lighthouse." aria-labelledby="fig12-caption" aria-describedby="fig13-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=167590779">
+  </a>
+  <div id="fig13-description" class="visually-hidden">Gráfico de barras que muestra  que en el percentil p10 0 ms pudieron ser medidos, lo mismo pasa en el percentil p25, en el percentil p50 se redujeron 150 ms, en el percentil p75 se redujeron 1.460 ms, y en el percentil p90 se redujeron 5.720 ms.</div>
+  <figcaption id="fig13-caption">Figura 13. Estimación de la mejora del rendimiento de la página tras la optimización de imagen de <i lang="en">Lighthouse</i>.</figcaption>
+</figure>
+
+### Imágenes adaptables (<i lang="en">responsive</i>)
```
Comment (bazzadp) [inline src/content/es/2019/media.md:243]: We generally don't put the `<i lang="en"></i>` elements in headings as it messes up the automatically created short links. See discussion here: #656 

```suggestion
### Imágenes adaptables
```

File: src/content/es/2019/media.md [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+  <a href="/static/images/2019/media/fig18_enabled_client_hints.png">
+    <img src="/static/images/2019/media/fig18_enabled_client_hints.png" alt="Figura 18. Client Hints habilitados." aria-labelledby="fig18-caption" aria-describedby="fig18-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=1878506264&format=interactive">
+  </a>
+  <div id="fig18-description" class="visually-hidden">Un gráfico circular de estilo donut que muestra que el 26,1% de los <i lang="en">client hints</i> usa 'dpr', 24,3% usa 'viewport-width', 19,7% usa 'width', 6,7% usa 'save-data', 6,1% usan 'device-memory', 6,0% usan 'downlink', 5,6% usan 'rtt' y 5,6% usan 'ect'</div>
+  <figcaption id="fig18-caption">Figura 18. <i lang="en">Client Hints</i> habilitados.</figcaption>
+</figure>
+
+Los <i lang="en">Client Hints</i> relacionados con la red, `downlink`, `rtt`, y `ect`, solamente están disponibles en Android Chrome.
+
+### <i lang="en">Lazy loading</i>
```
Comment (bazzadp) [inline src/content/es/2019/media.md:324]: We generally don't put the `<i lang="en"></i>` elements in headings as it messes up the automatically created short links. See discussion here: #656 

```suggestion
### Lazy loading
```

File: src/content/es/2019/media.md [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
+
+Para los dispositivos móviles usados en el rastreo (<i lang="en">crawl</i>) de páginas web, tenemos unas dimensiones de `512 x 360`, que suponen un 0,18 MP de contenido CSS (no confundir con las pantallas físicas para las cuales es `3x` o 3^2 píxeles más, 1,7 MP). Si se divide esta cantidad de píxeles del visor por el número de píxeles CSS correspondiente a las imágenes, obtenemos la cantidad de píxeles relativa.
+
+Si tuviéramos una imagen que ocupase perfectamente la totalidad de la pantalla, el ratio de relleno de píxeles sería de 1x. Por supuesto, rara vez una página web ocupa la pantalla completa con una única imagen. El contenido multimedia suele mezclarse con el diseño y otro tipo de contenido. Un valor superior a 1x significa que el diseño requiere que el usuario haga scroll para ver el resto de la imagen.   
+
+<p class="note">Nota: esto es únicamente analizando el diseño CSS para tanto el <i lang="en">viper</i> como para la cantidad de contenido del diseño. No está evaluándose la efectividad de las imágenes <i lang="en">responsive</i> o la efectividad de facilitar un contenido con alto DPR.</p>
```
Comment (bazzadp) [inline src/content/es/2019/media.md:81]: Authors confirmed in #1182 that it was a typo and should be DPR:
```suggestion
<p class="note">Nota: esto es únicamente analizando el diseño CSS para tanto el DPR como para la cantidad de contenido del diseño. No está evaluándose la efectividad de las imágenes <i lang="en">responsive</i> o la efectividad de facilitar un contenido con alto DPR.</p> 
```

**Comment by garcaplay** [comment]
Thanks for the review @bazzadp 
Going to make those changes asap and give an update here ;) (didn't thought on the urls when applied the replace all in the "video" word, my apologies for that)

**Comment by garcaplay** [comment]
@bazzadp branch updated (with the corresponding translator credit added this time). I've taken the liberty to change also the "viper" word to "DPR" in the English version.

**Pull request merged**